docker-compose.yml

```diff
@@ -75,7 +75,7 @@ services:
       - ASB_CONNECTION_STRING=some-azure-service-bus-connection-string
       - azure.application-insights.instrumentation-key=dummy
     ports:
-      - 8080:8080
+      - 8083:8080
     depends_on:
       - payments-database
       - service-auth-provider-api
```

Comment:
As discussed we want fees&pay applications, independent of s2s as well as idam. Hence please remove the dependency of service-auth-provider-api from payment API docker-compose.

Reply:
removed the service-auth-provider-api